src/Controls/src/Core/LegacyLayouts/Layout.cs

```diff
@@ -517,20 +527,13 @@ internal virtual void OnChildMeasureInvalidated(VisualElement child, Invalidatio
 				{
 					return;
 				}
-				if (trigger == InvalidationTrigger.HorizontalOptionsChanged || trigger == InvalidationTrigger.VerticalOptionsChanged)
+				if (trigger is InvalidationTrigger.HorizontalOptionsChanged or InvalidationTrigger.VerticalOptionsChanged)
 				{
 					ComputeConstraintForView(view);
```

Comment:
I just realized something..
`LegacyLayout` has this code which invalidates view legacy constraints with `ComputeConstraintForView`
But now, I'm not propagating MeasureInvalidated on *Options changes.
Which means this code will never run, and potentially generates wrong behavior.

So either:
- This entire method can be completely removed (if nowadays `ComputeContraintForView` is some kind of legacy XF code)
- My choice to not propagate `*Options` invalidation is wrong